src/sentry/tsdb/snuba.py

```diff
@@ -125,6 +108,12 @@ class SnubaTSDB(BaseTSDB):
         ),
     }
 
+    # ``model_columns_being_upgraded`` are models that currently use Redis but are being
+    # transitioned to use Snuba.
+    model_being_upgraded_query_settings = {}
+
+    model_being_upgraded_query_settings2 = {}
+
```

Comment:
These are referenced when deciding which models to write on redis, is that correct ?

Reply:
Yep!